ui/src/fields/Dictionary/Dictionary.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { DictionaryProps } from '@tesler-ui/core/components/ui/Dictionary/Dictionary'
+import { useSelector } from 'react-redux'
+import { AppState } from '../../interfaces/storeSlices'
+import { buildBcUrl } from '@tesler-ui/core'
+import styles from './Dictionary.module.css'
+import { RowMeta } from '@tesler-ui/core/interfaces/rowMeta'
+import { EMPTY_ARRAY } from '../../constants/constants'
+import { Dictionary as CoreDictionary } from '@tesler-ui/core'
+
+function Dictionary(props: DictionaryProps) {
+    const { value, meta, widgetName, backgroundColor } = props
+    const bcName = useSelector((state: AppState) => state.view.widgets?.find(i => i.name === widgetName)?.bcName)
+    const bcUrl = bcName && buildBcUrl(bcName, true)
+    const rowMeta = useSelector((state: AppState) => bcName && bcUrl && state.view.rowMeta[bcName]?.[bcUrl])
```

Comment:
@Dergash there is compilation error otherwise 
```
Type 'undefined' cannot be used as an index type.  TS2538

    13 |     const bcName = useSelector((state: AppState) => state.view.widgets?.find(i => i.name === widgetName)?.bcName)
    14 |     const bcUrl = bcName && buildBcUrl(bcName, true)
  > 15 |     const rowMeta = useSelector((state: AppState) => state.view.rowMeta[bcName]?.[bcUrl])
       |                                                                         ^
    16 |     const rowFieldMeta = (rowMeta as RowMeta)?.fields.find(field => field.key === meta?.key)
    17 |     if (meta?.bgColorKey) {
    18 |         return (

```